src/azure-cli/azure/cli/command_modules/appconfig/_constants.py

```diff
@@ -16,7 +16,12 @@ class FeatureFlagConstants:
 
 class KeyVaultConstants:
     KEYVAULT_CONTENT_TYPE = "application/vnd.microsoft.appconfig.keyvaultref+json;charset=utf-8"
+
+
+class AppServiceConstants:
+    APPSVC_CONFIG_REFERENCE_PREFIX = "@Microsoft.AppConfiguration("
```

Comment:
I just noticed that APPSVC_KEYVAULT_PREFIX doesn't end with `(`. Can we have AppConfig prefix without the ending bracket for consistency?